app.js

```diff
@@ -55,7 +55,7 @@ for(arg in process.argv) {
     case '-c' :
     case '--config' :
       configFile = __dirname + '/' + process.argv[parseInt(arg, 10) + 1];
-    break;
+      break;
```

Comment:
I actually prefer the "break" to remain unindented - despite what most style-checkers assert.  Switch statements are typically simple if/else statements, and the break helps distinguish the closing of a given condition.